sdk/src/realtime/composable.ts

```diff
@@ -26,6 +26,9 @@ const defaultRealTimeConfig: WebSocketConfig = {
 	},
 };
 
+const WS_OPEN = 1;
+const WS_CLOSED = 3;
```

Comment:
Shall we use a TS enum for this?

```ts
enum WebSocketState {
  Open = 1,
  Closed = 3
}

ws.readyState !== WebSocketState.Open
```

Reply:
I am not a huge fan of `enum`s as they walk the line between Type and Value resulting in some rough edges in their usage. I'd prefer a constant here since we don't need the Types side of the data.

```ts
const WebSocketState = {
  OPEN: 1,
  CLOSED: 3
} as const;
```

Reply:
I agree!

Reply:
Oki!